### PR TITLE
[staging] Backport ArtPiece layout and aspect ratio updates from BM

### PR DESCRIPTION
**Pull request by goran-peoski-work**
Continuation of
- https://github.com/sparkletown/sparkle/pull/2174
- https://github.com/sparkletown/sparkle/pull/2194

for
- https://github.com/sparkletown/internal-sparkle-issues/issues/1132

### CODE REVIEW

File: src/components/templates/ArtPiece/ArtPiece.tsx [hunk 1]
```diff
@@ -18,62 +20,83 @@ import SparkleFairiesPopUp from "components/molecules/SparkleFairiesPopUp/Sparkl
...
+  const { name, host, config, showRangers, iframeUrl, videoAspect } =
+    venue ?? {};
+
+  const landingPageConfig = config?.landingPageConfig;
+  const embeddableUrl = ConvertToEmbeddableUrl(iframeUrl);
 
-  const iframeUrl = ConvertToEmbeddableUrl(venue.iframeUrl);
+  const filteredAspect = filterAspectRatioProperty(videoAspect);
+  const customAspect = useCss({
+    "aspect-ratio": `${filteredAspect}`,
```
Comment (denisdimitrov) [inline src/components/templates/ArtPiece/ArtPiece.tsx:46]:
```suggestion
    "aspect-ratio": filteredAspect,
```

File: src/components/templates/ArtPiece/ArtPiece.tsx [hunk 2]
```diff
@@ -18,62 +20,83 @@ import SparkleFairiesPopUp from "components/molecules/SparkleFairiesPopUp/Sparkl
 
 import "./ArtPiece.scss";
 
+const DECLARATIVE_ASPECT_RATIOS = [
+  `${VideoAspectRatio.sixteenNine}`,
+  `${VideoAspectRatio.anamorphic}`,
+  `${VideoAspectRatio.fullWidth}`,
```
Comment (denisdimitrov) [inline src/components/templates/ArtPiece/ArtPiece.tsx:26]: No need for `${}`

**Review by denisdimitrov — APPROVED** [review]
LGTM!

File: src/components/templates/ArtPiece/ArtPiece.tsx (hunk 2 above)
Comment (goran-peoski-work) [reply src/components/templates/ArtPiece/ArtPiece.tsx:26]: Yes there is. TS complains enums aren't strings, so a bit of forcing our or its hand...

**Pull request merged**